workshops/particle_physics/README.md

```diff
@@ -21,10 +21,51 @@ This workshop should take 15-20 minutes to complete.
 
 ## Getting Set Up
 
+# Github Codespaces
+
+This workshop will be using [Github Codespaces](https://github.com/features/codespaces), an online code editor.
+
+To get started, make a Github account (or login if you have one) at [https://github.com/signup](https://github.com/signup).
+
+# Project Setup
+
 First you need to setup your project. I created a basic p5.js [boilerplate](https://en.wikipedia.org/wiki/Boilerplate_code) that you can use to easily start coding.
 
-1. Go to [https://repl.it/@SquarePear/p5-boilerplate](https://repl.it/@SquarePear/p5-boilerplate)
-2. Click "fork" to make a copy for you to edit
+1. Go to [https://repl.it/@SquarePear/p5-boilerplate](https://github.com/Luna5379/p5-boilerplate/fork)
```

Comment:
shouldn't the link text be updated as well? could be misleading to users

Reply:
ah yes, sorry i forgot ill do that now!

Reply:
done!